src/react-app-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="react-scripts" />
```

Comment:
Do czego jest ten plik?

Reply:
Usuwałam go już ze dwa razy, jak się przełączyłam na nowy branch to znowu mam ten plik :/